index.html

```diff
@@ -18,18 +18,31 @@
 
 <body>
 
-  <main class="main">
-    <header class="header">
-      <nav class="nav-bar">
-        <a href="#" class="my-logo">CAkE</a>
-        <span class="mob-menu"><img src="Images&Icons/Enabled.png" alt="" /></span>
-        <ul class="desktop-nav">
-          <li><a href="">Portfolio</a></li>
-          <li><a href="">About</a></li>
-          <li><a href="">Contact</a></li>
+  <header>
+    <div class="container">

```

Comment:
- [ optional] the menu might have disrupted the responsiveness of your portfolio. Please look into that
![Screenshot (254)_LI](https://user-images.githubusercontent.com/65298048/179803556-20236156-24b7-4520-8c28-6c59682386c9.jpg)